tests/test_masks.py

```diff
@@ -134,26 +134,65 @@ def test_shrink(self):
 
     def test_shrink_nd(self):
         size = 3
-        mp = Mask((size, size - 1))
+        mp = DropoutMask((size, size - 1))
         for amount in range(mp.mask.size + 1):
             mp[:] = True
             mp.shrink(amount)
             assert pnp.sum(mp.mask) == mp.mask.size - amount
 
     def test_copy(self):
         size = 3
-        mp = Mask((size,))
+        mp = DropoutMask((size,))
         new_mp = mp.copy()
         mp[0] = True
         assert pnp.sum(mp.mask) > pnp.sum(new_mp.mask)
         assert pnp.sum(new_mp.mask) == 0
 
     def test_apply_mask(self):
         size = 3
-        mp = Mask((size,))
+        mp = DropoutMask((size,))
         with pytest.raises(IndexError):
             mp.apply_mask(pnp.ones((size - 1, size)))
         mp.mask[1] = True
         result = mp.apply_mask(pnp.ones((size,), dtype=bool))
         assert pnp.sum(mp.mask) == 1
         assert pnp.sum(result) == size - 1
+
+
+class TestValueMask:
+    def test_init(self):
+        vm = ValueMask((3,))
+        assert vm
```

Comment:
``ValueMask`` doesn't seem to implement ``__bool__``. I recommend not to test the implicit ``__bool__`` (which is always ``True``) – it doesn't say much but can stealthily lock in the API.